### DISCUSSION TITLE
Experimental media-settings-* files are missing from the npm package

### DISCUSSION THREAD

**Discussion by moaali** [muxinc/media-chrome#806] [Q&A] [answered]
Some files from the `experimental` and `experimental/menu` folders are missing from the npm published package.

Like:

- media-settings-menu-button.js
- media-settings-menu-item.js
- media-settings-menu.js
- media-chrome-dialog.js

**Comment by cjpillsbury** [comment]
uh oh let us take a look, @moaali. Thanks for the heads up!

**Comment by cjpillsbury** [comment, marked as answer]
@moaali it looks like they're in our latest canary release (https://www.npmjs.com/package/media-chrome/v/2.1.1-canary.10-545d9b8) and _**should**_ be in our next official release.
